org.eclipse.lsp4xml/src/main/java/org/eclipse/lsp4xml/extensions/contentmodel/participants/diagnostics/LSPXMLGrammarPool.java

```diff
@@ -0,0 +1,419 @@
+/*

```

Comment:
where is this file coming from? need to document it

Reply:
fixed